docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md

```diff
@@ -0,0 +1,218 @@
+---
+description: Dismounts an assigned device from a virtual machine host.
```

Comment:
Suggestion: "Dismounts an **assignable** device..."

The device shouldn't be assigned anywhere at this point, these commandlets are for managing the devices on the host before assigning (or after assigning)